test/functional/ex_cmds/wviminfo_spec.lua

```diff
@@ -36,7 +37,8 @@ describe(':wshada', function()
     write_file(shada_file, text)
 
     -- sanity check
-    eq(text, io.open(shada_file):read())
+    local fp0 = io.open(shada_file)
+    eq(text, fp0:read())
```

Comment:
`helpers.read_file()` does the open-read-close in one step. Or must the close be done at the end where you added it below?

Reply:
I assumed that the order is important and I haven't used `:wshada` before. Does the test change if I use `helpers.read_file()`?

Reply:
@janlazo good thought, but this was accidental. 

    eq(text, read_file(shada_file))

Reply:
Can I do the same for the`fp:read()` after `:wshada`?

Reply:
@janlazo I would leave it as-is since it's specifically getting only the first char.